actionview/RUNNING_UJS_TESTS.rdoc

```diff
@@ -1,8 +1,13 @@
 == Running UJS tests
 
-Ensure that you can build the project by running:
+Run the tests in headless mode by running:
+
+   rake test:ujs
+
+To run the tests in a browser, start the Rails UJS server by running:
+
    rake ujs:server
 
-Then run the web tests by visiting the following URL in your browser:
+Then, in a separate tab, start the Karma server by running:
 
-   http://localhost:4567
+   karma start karma.config.js --no-single-run --browsers Chrome
```

Comment:
Maybe we should wrap this in a task so no one has to remember or copy and paste this much. Something like `rake karma:start`?  Or make it agnostic so we can swap out in the future `rake test:start_browser`

Reply:
I have now combined this into the `rake ujs:server` task so it will start the server and then start the karma runner which opens the browser.